src/CalcViewModel/StandardCalculatorViewModel.cpp

```diff
@@ -1016,7 +1013,7 @@ void StandardCalculatorViewModel::OnMemoryButtonPressed()
     m_standardCalculatorManager.MemorizeNumber();
 
     int windowId = Utils::GetWindowId();
-    TraceLogger::GetInstance().InsertIntoMemoryMap(windowId, IsStandard, IsScientific, IsProgrammer);
+    TraceLogger::GetInstance().InsertIntoMemoryMap(windowId, IsScientific, IsProgrammer);
```

Comment:
Instead of removing `IsStandard`, we should update `InsertIntoMemoryMap` to use the bool.

Reply:
That param is not needed; looking in the function you'll see that it defaults to Standard if it's not Scientific or Programmer mode.

Reply:
I understand.  I'm asking that we update the TraceLogger function to use the bool as part of an `else if`.  Then if all three are false, the function will no-op.  The current change would default to assuming Standard, which isn't necessarily correct.

Reply:
got it.  Updated.